lib/assets/javascripts/turbograft/page.coffee

```diff
@@ -19,9 +19,12 @@ Page.refresh = (options = {}, callback) ->
   if options.response
     onlyKeys   = options.onlyKeys   || []
     exceptKeys = options.exceptKeys || []
-    Turbolinks.loadPage null, options.response, true, callback, onlyKeys, exceptKeys
+    Turbolinks.loadPage null, options.response, true, true, callback, onlyKeys, exceptKeys
```

Comment:
I say only slightly better because it gets messy dealing with default values for arguments when they are not part of the signature.